x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/terms/index.tsx

```diff
@@ -316,9 +317,25 @@ export const termsOperation: OperationDefinition<TermsIndexPatternColumn, 'field
           </EuiText>
         )}
         <EuiFormRow
-          label={i18n.translate('xpack.lens.indexPattern.terms.orderBy', {
-            defaultMessage: 'Order by',
-          })}
+          label={
+            <>
+              {i18n.translate('xpack.lens.indexPattern.terms.orderBy', {
+                defaultMessage: 'Rank by',
+              })}{' '}
+              <EuiIconTip
+                color="subdued"
+                content={i18n.translate('xpack.lens.indexPattern.terms.orderByHelp', {
+                  defaultMessage: `It controls the ranking system used to retrieve the Top values.`,
```

Comment:
@KOTungseth Should "Top values" be capitalized here? I'm always hazy on these rules.

Reply:
How about:

`Specifies the function that retrieves the top values.`

Since we are referring to the top values in the data set instead of the UI setting, `top values` can be lowercase.